source/stream.js

```diff
@@ -0,0 +1,62 @@
+import {isReadableStream} from 'is-stream';
+
+export const getAsyncIterable = stream => {
+	if (isReadableStream(stream, {checkOpen: false})) {
+		return getStreamIterable(stream);
+	}
+
+	if (stream?.[Symbol.asyncIterator] === undefined) {
+		throw new Error('The first argument must be a Readable, a ReadableStream, or an async iterable.');
+	}
+
+	return stream;
+};
+
+// The default iterable for Node.js streams does not allow for multiple readers at once, so we re-implement it
+const getStreamIterable = async function * (stream) {
+	if (nodeImports === undefined) {
+		await loadNodeImports();
+	}
+
+	const controller = new AbortController();
+	handleStreamEnd(stream, controller);
+
+	try {
+		for await (const [chunk] of nodeImports.events.on(stream, 'data', {
+			signal: controller.signal,
+			highWatermark: stream.readableHighWaterMark,
```

Comment:
That option is undocumented but it is quite nifty. It was added by https://github.com/nodejs/node/pull/41276 for `readline` `Interface`. 

I have opened https://github.com/nodejs/node/issues/52078 since I'm not sure it was left undocumented on purpose.

Regardless, even if the option disappeared, this option is only meant to reduce memory consumption (by automatically pausing/resuming the stream when too much data is buffering), so it would not break the usage.

Reply:
Apparently, they are already working on it! :tada: 

https://github.com/nodejs/node/pull/52080